lib/src/components/NxFilterInput/NxFilterInput.scss

```diff
@@ -7,71 +7,46 @@
 @import '../../scss-shared/nx-variables';
 @import '../../scss-shared/nx-container-helpers';
 
-$nx-filter-input-height: 20px;
-
 .nx-filter-input {
   @include container-horizontal;
 
   display: inline-block;
-  vertical-align: middle;
   white-space: nowrap;
 
-  .nx-filter-text-input {
-    border: $nx-border;
-    border-left: 0;
-    font-size: 14px;
-    height: $nx-filter-input-height;
-    margin-left: 0;
-    padding: 4px 6px;
-    box-sizing: content-box;
+  &:focus {
+    outline: 0;
```

Comment:
I'm fairly sure this doesn't do anything; the div isn't going to be getting focus

Reply:
> I'm fairly sure this doesn't do anything; the div isn't going to be getting focus

https://github.com/sonatype/sonatype-react-shared-components/pull/62/commits/0c354249c5b6f29decf837b6f576ff3ddf5068cc